example/index.js

```diff
@@ -4,8 +4,9 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { initialize, getConfig, subscribe, APP_READY } from '@edx/frontend-platform';
 import { AppContext, AppProvider } from '@edx/frontend-platform/react';
+import Header from '@edx/frontend-component-header';
```

Comment:
webpack.dev.config.js is configured to use @edx/frontend-component-header as an alias for the `src` directory.